### PR TITLE
Fix the Errors/Warnings when building Qlib's documentation

### LINKED ISSUE #1345: Fix the Errors/Warnings  when building Qlib's documentation.
## 📖 Documentation

image

### PR DESCRIPTION
**Pull request by MaximSmolskiy**
## Description

Fix all the Errors/Warnings when building Qlib's documentation

## Motivation and Context

Fix #1345

## How Has This Been Tested?

- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [x] Update documentation

### CODE REVIEW

**Comment by MaximSmolskiy** [comment]
@you-n-g please check that this pr fixes all warnings in your environment and finally fixes the issue

**Review by you-n-g — COMMENTED** [review]
It looks great! 
All warnings disappeared.

I think it would be better if we can check the doc compiling in the CI to protect current fruits.

**Comment by MaximSmolskiy** [comment]
> It looks great! All warnings have disappeared.
> 
> I think it would be better if we can check the doc compiling in the CI to protect current fruits.

@you-n-g Added doc compiling checking to CI and tested that it works

File: .github/workflows/test_qlib_from_source.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
     - name: Make html with sphinx
       run: |
         cd docs 
-        sphinx-build -b html . build
+        sphinx-build -W --keep-going -b html . build
```
Comment (you-n-g) [inline .github/workflows/test_qlib_from_source.yml:63]:
```suggestion
        sphinx-build -W --keep-going -b html . _build
```

Comment (you-n-g) [reply .github/workflows/test_qlib_from_source.yml:63]: Tries to be consistent with Makefile

**Comment by you-n-g** [comment]
Thank you for making the docs great.

**Pull request merged**